api_tests/src/actors/order_factory.ts

```diff
@@ -29,15 +29,16 @@ interface Ethereum {
 
 export default class OrderbookFactory {
     public static async connect(alice: Actor, bob: Actor) {
-        // Get alice's listen address
-        const aliceAddr = await alice.cnd.getPeerListenAddresses();
-
-        // Bob dials alices
+        // Alice dials Bob, must be this way around in order to ensure
+        // Alice subscribes to the BTC/DAI topic _after_ she has a connection.
+        const addr = await bob.cnd.getPeerListenAddresses();
         // @ts-ignore
```

Comment:
It's always good to justify the `@ts-ignore` so one can quickly whether it still applies.

Reply:
Good point, thanks. I have been asking if these ts-ignores are needed since this work started but got no answer.

Reply:
Remove them and see if you can run the e2e tests :) but yeah, need to justify exactly for the reason you are saying! Maybe they are here because the types of an upstream lib are not up to date, meaning they can be removed one the upstream lib is updated.
Or maybe they are here because we are breaking the type system in purpose, in this case it's good to explain why.

Reply:
Cool thanks, fixed: https://github.com/comit-network/comit-rs/pull/3009